pipelines/managed/rh-advisories/rh-advisories.yaml

```diff
@@ -349,6 +351,7 @@ spec:
           value: "$(params.taskGitRevision)"
       runAfter:
         - apply-mapping
+        - collect-data
```

Comment:
apply-mapping happens after collect-data anyway, so this is probably not needed? But this uses `filter-already-released-advisory-images` for `sourceDataArtifact` which happens after both of them. So I'd say if anything it would make sense to include `filter-already-released-advisory-images` in runAfter?